src/composable/orderTypes/Twap.ts

```diff
@@ -363,9 +389,11 @@ export class Twap extends ConditionalOrder<TwapData, TwapStruct> {
     return {
       result: PollResultCode.TRY_AT_EPOCH,
       epoch: nextPartStartTime,
-      reason: `Current active TWAP part (${currentPartNumber}/${numberOfParts}) is already in the Order Book. TWAP part ${currentPartNumber} doesn't start until ${nextPartStartTime} (${formatEpoch(
-        nextPartStartTime
-      )})`,
+      reason: `Current active TWAP part (${
+        currentPartNumber + 1
+      }/${numberOfParts}) is already in the Order Book. TWAP part ${
+        currentPartNumber + 2
+      } doesn't start until ${nextPartStartTime} (${formatEpoch(nextPartStartTime)})`,
```

Comment:
Given debugging / logs here, I'd suggest that we make notes that TWAPs are indexed by '0', but logged with indexes commencing at 1. Alternatively (and one that may be simpler from a debugging perspective) is to report the part as zero indexed, and explicitly state the total. Something like:

`Current active TWAP part 0[5] is already in the Order Book. TWAP part 1[5] doesn't start until` etc. This would have the debugging messages remain consistent with what developers can expect (and potentially prevent silly off-by-one issues).

I have no strong opinion as to which one to prefer, just that we should either establish a standard (not sure if we have a standard in place for this already?), or keep logging indexes the same as that used within code.

Reply:
Good point, i will try to add some comments. I mean, the reason is just a human readable message, i see your point on being 0-indexed, but the SDK is offering some easier way to use the TWAP, this is why  it lets the user use`TwapData` instead of making them work with the `TwapStruct`, so my thinking is that is easier to understand 1-index based twap orders: there's 10 orders, 1/10, 2/10  is simpler than there's 10 orders 0/9 which is the first one, 1/9 which is the second one. 

This is just a message to see in the logs, so I think is fine, I will document is not 0-indexed though.

Let me know if you feel strong about this one